examples/vertexai/bootstrap.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+use Google\Auth\ApplicationDefaultCredentials;
```

Comment:
this needs a new require in the `examples/composer.json` file - cannot run the examples locally at this point, sorry